client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -273,6 +273,9 @@ public class ClientOptions
     @Option(names = "--disable-auto-suggestion", description = "Disable auto suggestion")
     public boolean disableAutoSuggestion;
 
+    @Option(names = "--follow-redirects", description = "Follow client redirects from server")
```

Comment:
Since redirect is default true, we might want a flag to disable the it.
Following the current naming convention, I suggest `--disable-follow-redirect`.

Reply:
Ack.